cmd/quarterdeck/main.go

```diff
@@ -219,7 +240,93 @@ func listOrgs(c *cli.Context) (err error) {
 	return nil
 }
 
-func cleanupUsers(c *cli.Context) (err error) {
+func cleanupOrgs(c *cli.Context) (errs error) {
```

Comment:
Theoretically stranded organizations won't happen unless something went wrong because the users cleanup command now also removes the organization.